icu4c/source/data/buildtool/filtration.py

```diff
@@ -164,15 +164,16 @@ def __init__(self, json_data):
     def _add_locale_and_parents(self, locale):
         # Store the locale as *requested*
         self.locales_requested.add(locale)
-        # Store the locale's dependencies as *required*
+        # Store the locale and its dependencies as *required*
         while locale is not None:
             self.locales_required.add(locale)
             locale = self._get_parent_locale(locale)
 
     def match(self, file):
         locale = self._file_to_file_stem(file)
 
-        # A locale is *required* if it is a parent of a *requested* locale.
+        # A locale is *required* if it is *requested* or a parent of a
```

Comment:
I tried to make it more explicit that it can be a parent, grandparent, great-grandparent...
Maybe "... or an ancestor of ..."?

Reply:
Done.